plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java

```diff
@@ -389,6 +392,9 @@ private ILaunch attachRemoteDebugger(IServer server, int localDebugPort, IProgre
 		monitor.subTask("Attaching remote debugger...");
 		ILaunch ret = null;
 		DebugLaunchConfigs launchConfigs = DebugLaunchConfigs.get();
+		if( launchConfigs == null ) {
+			throw toCoreException(NLS.bind("Could not modify launch config for server {0}", server.getName()));
```

Comment:
It should be `Could not get launch config for server {0} to attach remote debugger`, should it not? This way it would be in some way related to action 'Attaching remote debugger' and it would by obvious why it cannot be attached.